modules/metrics/src/main/java/org/apache/ignite/internal/metrics/MetricManager.java

```diff
@@ -178,15 +192,19 @@ public static Map<String, MetricExporter> loadExporters() {
      */
     public void disable(MetricSource src) {
         registry.disable(src);
+
+        enabledMetricExporters.values().forEach(e -> e.removeMetricSet(src.name()));
     }
 
     /**
-     * Disable metric source by name. See {@link MetricRegistry#disable(String)}.
+     * Disable metric set by name. See {@link MetricRegistry#disable(String)}.
      *
-     * @param srcName Source name.
+     * @param metricSetName Metric set name.
      */
-    public void disable(final String srcName) {
-        registry.disable(srcName);
+    public void disable(final String metricSetName) {
```

Comment:
I don't completely understand this change. It refers to `MetricRegistry#disable(String)` but it disables metric source by its name. Also, it brings inconsistency to API, as `MetricManager#enable(String)` enables metric source, not metric set. Finally, only metric source can be enabled/disabled. Could you explain?

Reply:
Renamed by mistake. Fixed, thanks.